integration-tests/src/test/java/gov/uk/address/api/stepDefinitions/AddressApiHappyPath.java

```diff
@@ -355,6 +355,15 @@ public void userShouldReceiveAValidResponse() throws ParseException, IOException
                 payload.get("result").get("addresses").get(0).get("postalCode").asText());
     }
 
+    @And("JWT lives for two hours")
+    public void jwt_lives_for_two_hours() throws ParseException, IOException {
+        SignedJWT decodedJWT = SignedJWT.parse(response.body());
+        var payload = objectMapper.readTree(decodedJWT.getPayload().toString());
+        assertNotNull(payload.get("nbf"));
+        assertNotNull(payload.get("exp"));
+        assertEquals(7200L, payload.get("exp").asLong() - payload.get("nbf").asLong());
+    }
```

Comment:
Is it worthwhile moving the above assertions to the `makeAssertions` method below? This way we can remove this method and the extra line in the feature file 'And JWT lives for two hours'